skopeo.spec.rpkg

```diff
@@ -0,0 +1,126 @@
+# For automatic rebuilds in COPR
```

Comment:
A totally non-blocking it: Would it be (_without extra hassle_) possible to move this somewhere to `contrib`? We don’t really want users to build their own RPMs, and that’s where the container image descriptions are (admittedly `contrib` is not a great name for “our own build infrastructure that other users shouldn’t need to touch”).

OTOH I see `podman.spec.rpkg` in the same place, and that consistency is certainly valuable. So entirely up to you.

Reply:
Let me check if that's doable. IIRC, this file is expected in the main dir, but let me confirm..